contracts/DutchExchange.sol

```diff
@@ -680,15 +673,18 @@ contract DutchExchange {
 
         // if (opposite is zero auction OR opposite auction has cleared) {
         if ((sellVolumesCurrent[buyToken][sellToken] == 0 && opp.den == 0) || opp.den > 0) {
```

Comment:
I don't think we need opp.den == 0?

Reply:
other than that, loooks good.

Reply:
in fact, it isn't even logically correct. If opp.den > 0, the second condition will fire ...

Reply:
I agree, we do not need it. I added it, because it increases readability:
(sellVolumesCurrent[buyToken][sellToken] == 0 && opp.den == 0)==> dealing with 0 sell order
opp.den > 0 ==>opposite auction was already closed.

Reply:
anyway, I am okay with taking && opp.den out.